packages/wallets/src/features/cfd/screens/DynamicLeverage/DynamicLeverageTableColumnHeader.scss

```
@@ -0,0 +1,12 @@
+.wallets-dynamic-leverage-modal {
+    &__market {
+        &-table {
+            &-header {
+                &-cell {
+                    display: flex;
+                    flex-direction: column;
+                }
+            }
```

Since its only one variant, maybe we can combine these together?
```suggestion
         &__market-table-header-cell {}
```

---

Updated.